### ISSUE TITLE
DEPRECATION WARNING in Ruby On Rails + React

### ISSUE THREAD

**Issue by andrijzyn** [twbs/bootstrap#41071]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

When initializing the RoR + React example, there is a problem with Bootstrap support

### Reduced test cases

```
❯ bin/dev             
15:22:02 web.1  | started with pid 224822
15:22:02 js.1   | started with pid 224823
15:22:02 css.1  | started with pid 224824
15:22:03 js.1   | yarn run v1.22.22
15:22:03 css.1  | yarn run v1.22.22
15:22:03 js.1   | warning ../../package.json: No license field
15:22:03 css.1  | warning ../../package.json: No license field
15:22:03 js.1   | $ esbuild app/javascript/*.* --bundle --sourcemap --format=esm --outdir=app/assets/builds --public-path=/assets --watch
15:22:03 css.1  | $ nodemon --watch ./app/assets/stylesheets/ --ext scss --exec "yarn build:css"
15:22:03 css.1  | [nodemon] 3.1.7
15:22:03 css.1  | [nodemon] to restart at any time, enter `rs`
15:22:03 css.1  | [nodemon] watching path(s): app/assets/stylesheets/**/*
15:22:03 css.1  | [nodemon] watching extensions: scss
15:22:03 css.1  | [nodemon] starting `yarn build:css`
15:22:03 js.1   | [watch] build finished, watching for changes...
15:22:04 css.1  | warning ../../package.json: No license field
15:22:04 css.1  | $ yarn build:css:compile && yarn build:css:prefix
15:22:04 web.1  | => Booting Puma
15:22:04 web.1  | => Rails 8.0.0 application starting in development 
15:22:04 web.1  | => Run `bin/rails server --help` for more startup options
15:22:04 css.1  | warning ../../package.json: No license field
15:22:04 css.1  | $ sass ./app/assets/stylesheets/application.bootstrap.scss:./app/assets/builds/application.css --no-source-map --load-path=node_modules
15:22:04 web.1  | Puma starting in single mode...
15:22:04 web.1  | * Puma version: 6.5.0 ("Sky's Version")
15:22:04 web.1  | * Ruby version: ruby 3.3.6 (2024-11-05 revision 75015d4c1f) [x86_64-linux]
15:22:04 web.1  | *  Min threads: 3
15:22:04 web.1  | *  Max threads: 3
15:22:04 web.1  | *  Environment: development
15:22:04 web.1  | *          PID: 224822
15:22:04 web.1  | * Listening on http://127.0.0.1:3000
15:22:04 web.1  | * Listening on http://[::1]:3000
15:22:04 web.1  | Use Ctrl-C to stop
15:22:05 css.1  | DEPRECATION WARNING: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |   ╷
15:22:05 css.1  | 1 │ @import 'bootstrap/scss/bootstrap';
15:22:05 css.1  |   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^
15:22:05 css.1  |   ╵
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |   ╷
15:22:05 css.1  | 2 │ @import 'bootstrap-icons/font/bootstrap-icons';
15:22:05 css.1  |   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
15:22:05 css.1  |   ╵
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 2:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |   ╷
15:22:05 css.1  | 1 │ @import "mixins/banner";
15:22:05 css.1  |   │         ^^^^^^^^^^^^^^^
15:22:05 css.1  |   ╵
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 1:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |   ╷
15:22:05 css.1  | 7 │ @import "functions";
15:22:05 css.1  |   │         ^^^^^^^^^^^
15:22:05 css.1  |   ╵
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 7:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |   ╷
15:22:05 css.1  | 8 │ @import "variables";
15:22:05 css.1  |   │         ^^^^^^^^^^^
15:22:05 css.1  |   ╵
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Global built-in functions are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | Use color.mix instead.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 207 │   @return mix(white, $color, $weight);
15:22:05 css.1  |     │           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 207:11     tint-color()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 79:12      @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Global built-in functions are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | Use color.mix instead.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 212 │   @return mix(black, $color, $weight);
15:22:05 css.1  |     │           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 212:11     shade-color()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 84:12      @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Global built-in functions are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | Use color.mix instead.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 342 │ $light-bg-subtle:         mix($gray-100, $white) !default;
15:22:05 css.1  |     │                           ^^^^^^^^^^^^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 342:27     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Global built-in functions are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | Use math.unit instead.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |    ╷
15:22:05 css.1  | 11 │     @if $prev-num == null or unit($num) == "%" or unit($prev-num) == "%" {
15:22:05 css.1  |    │                              ^^^^^^^^^^
15:22:05 css.1  |    ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 11:30      -assert-ascending()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 494:1      @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: Global built-in functions are deprecated and will be removed in Dart Sass 3.0.0.
15:22:05 css.1  | Use math.unit instead.
15:22:05 css.1  | 
15:22:05 css.1  | More info and automated migrator: https://sass-lang.com/d/import
15:22:05 css.1  | 
15:22:05 css.1  |    ╷
15:22:05 css.1  | 11 │     @if $prev-num == null or unit($num) == "%" or unit($prev-num) == "%" {
15:22:05 css.1  |    │                                                   ^^^^^^^^^^^^^^^
15:22:05 css.1  |    ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 11:51      -assert-ascending()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 494:1      @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: red() is deprecated. Suggestion:
15:22:05 css.1  | 
15:22:05 css.1  | color.channel($color, "red", $space: rgb)
15:22:05 css.1  | 
15:22:05 css.1  | More info: https://sass-lang.com/d/color-functions
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 185 │     "r": red($color),
15:22:05 css.1  |     │          ^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 185:10     luminance()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 174:8      contrast-ratio()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 159:22     color-contrast()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 846:42     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: green() is deprecated. Suggestion:
15:22:05 css.1  | 
15:22:05 css.1  | color.channel($color, "green", $space: rgb)
15:22:05 css.1  | 
15:22:05 css.1  | More info: https://sass-lang.com/d/color-functions
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 186 │     "g": green($color),
15:22:05 css.1  |     │          ^^^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 186:10     luminance()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 174:8      contrast-ratio()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 159:22     color-contrast()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 846:42     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: blue() is deprecated. Suggestion:
15:22:05 css.1  | 
15:22:05 css.1  | color.channel($color, "blue", $space: rgb)
15:22:05 css.1  | 
15:22:05 css.1  | More info: https://sass-lang.com/d/color-functions
15:22:05 css.1  | 
15:22:05 css.1  |     ╷
15:22:05 css.1  | 187 │     "b": blue($color)
15:22:05 css.1  |     │          ^^^^^^^^^^^^
15:22:05 css.1  |     ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 187:10     luminance()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 174:8      contrast-ratio()
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 159:22     color-contrast()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 846:42     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: red() is deprecated. Suggestion:
15:22:05 css.1  | 
15:22:05 css.1  | color.channel($color, "red", $space: rgb)
15:22:05 css.1  | 
15:22:05 css.1  | More info: https://sass-lang.com/d/color-functions
15:22:05 css.1  | 
15:22:05 css.1  |    ╷
15:22:05 css.1  | 37 │   @return red($value), green($value), blue($value);
15:22:05 css.1  |    │           ^^^^^^^^^^^
15:22:05 css.1  |    ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 37:11      to-rgb()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 846:31     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:05 css.1  | DEPRECATION WARNING: green() is deprecated. Suggestion:
15:22:05 css.1  | 
15:22:05 css.1  | color.channel($color, "green", $space: rgb)
15:22:05 css.1  | 
15:22:05 css.1  | More info: https://sass-lang.com/d/color-functions
15:22:05 css.1  | 
15:22:05 css.1  |    ╷
15:22:05 css.1  | 37 │   @return red($value), green($value), blue($value);
15:22:05 css.1  |    │                        ^^^^^^^^^^^^^
15:22:05 css.1  |    ╵
15:22:05 css.1  |     node_modules/bootstrap/scss/_functions.scss 37:24      to-rgb()
15:22:05 css.1  |     node_modules/bootstrap/scss/_variables.scss 846:31     @import
15:22:05 css.1  |     bootstrap/scss/bootstrap.scss 8:9                      @import
15:22:05 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:05 css.1  | 
15:22:06 css.1  | DEPRECATION WARNING: Sass's behavior for declarations that appear after nested
15:22:06 css.1  | rules will be changing to match the behavior specified by CSS in an upcoming
15:22:06 css.1  | version. To keep the existing behavior, move the declaration above the nested
15:22:06 css.1  | rule. To opt into the new behavior, wrap the declaration in `& {}`.
15:22:06 css.1  | 
15:22:06 css.1  | More info: https://sass-lang.com/d/mixed-decls
15:22:06 css.1  | 
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/_reboot.scss
15:22:06 css.1  | 503 │     font-weight: $legend-font-weight;
15:22:06 css.1  |     │     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
15:22:06 css.1  |     ╵
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/vendor/_rfs.scss
15:22:06 css.1  | 136 │ ┌     @media (#{$rfs-mq-property-width}: #{$rfs-mq-value}) {
15:22:06 css.1  | 137 │ │       @content;
15:22:06 css.1  | 138 │ │     }
15:22:06 css.1  |     │ └─── nested rule
15:22:06 css.1  |     ╵
15:22:06 css.1  |     node_modules/bootstrap/scss/_reboot.scss 503:3         @import
15:22:06 css.1  |     bootstrap/scss/bootstrap.scss 16:9                     @import
15:22:06 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:06 css.1  | 
15:22:06 css.1  | DEPRECATION WARNING: Sass's behavior for declarations that appear after nested
15:22:06 css.1  | rules will be changing to match the behavior specified by CSS in an upcoming
15:22:06 css.1  | version. To keep the existing behavior, move the declaration above the nested
15:22:06 css.1  | rule. To opt into the new behavior, wrap the declaration in `& {}`.
15:22:06 css.1  | 
15:22:06 css.1  | More info: https://sass-lang.com/d/mixed-decls
15:22:06 css.1  | 
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/_reboot.scss
15:22:06 css.1  | 504 │     line-height: inherit;
15:22:06 css.1  |     │     ^^^^^^^^^^^^^^^^^^^^ declaration
15:22:06 css.1  |     ╵
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/vendor/_rfs.scss
15:22:06 css.1  | 136 │ ┌     @media (#{$rfs-mq-property-width}: #{$rfs-mq-value}) {
15:22:06 css.1  | 137 │ │       @content;
15:22:06 css.1  | 138 │ │     }
15:22:06 css.1  |     │ └─── nested rule
15:22:06 css.1  |     ╵
15:22:06 css.1  |     node_modules/bootstrap/scss/_reboot.scss 504:3         @import
15:22:06 css.1  |     bootstrap/scss/bootstrap.scss 16:9                     @import
15:22:06 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:06 css.1  | 
15:22:06 css.1  | DEPRECATION WARNING: Sass's behavior for declarations that appear after nested
15:22:06 css.1  | rules will be changing to match the behavior specified by CSS in an upcoming
15:22:06 css.1  | version. To keep the existing behavior, move the declaration above the nested
15:22:06 css.1  | rule. To opt into the new behavior, wrap the declaration in `& {}`.
15:22:06 css.1  | 
15:22:06 css.1  | More info: https://sass-lang.com/d/mixed-decls
15:22:06 css.1  | 
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/_type.scss
15:22:06 css.1  | 38  │       font-family: $display-font-family;
15:22:06 css.1  |     │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
15:22:06 css.1  |     ╵
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/vendor/_rfs.scss
15:22:06 css.1  | 136 │ ┌     @media (#{$rfs-mq-property-width}: #{$rfs-mq-value}) {
15:22:06 css.1  | 137 │ │       @content;
15:22:06 css.1  | 138 │ │     }
15:22:06 css.1  |     │ └─── nested rule
15:22:06 css.1  |     ╵
15:22:06 css.1  |     node_modules/bootstrap/scss/_type.scss 38:5            @import
15:22:06 css.1  |     bootstrap/scss/bootstrap.scss 17:9                     @import
15:22:06 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:06 css.1  | 
15:22:06 css.1  | DEPRECATION WARNING: Sass's behavior for declarations that appear after nested
15:22:06 css.1  | rules will be changing to match the behavior specified by CSS in an upcoming
15:22:06 css.1  | version. To keep the existing behavior, move the declaration above the nested
15:22:06 css.1  | rule. To opt into the new behavior, wrap the declaration in `& {}`.
15:22:06 css.1  | 
15:22:06 css.1  | More info: https://sass-lang.com/d/mixed-decls
15:22:06 css.1  | 
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/_type.scss
15:22:06 css.1  | 39  │       font-style: $display-font-style;
15:22:06 css.1  |     │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
15:22:06 css.1  |     ╵
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/vendor/_rfs.scss
15:22:06 css.1  | 136 │ ┌     @media (#{$rfs-mq-property-width}: #{$rfs-mq-value}) {
15:22:06 css.1  | 137 │ │       @content;
15:22:06 css.1  | 138 │ │     }
15:22:06 css.1  |     │ └─── nested rule
15:22:06 css.1  |     ╵
15:22:06 css.1  |     node_modules/bootstrap/scss/_type.scss 39:5            @import
15:22:06 css.1  |     bootstrap/scss/bootstrap.scss 17:9                     @import
15:22:06 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:06 css.1  | 
15:22:06 css.1  | DEPRECATION WARNING: Sass's behavior for declarations that appear after nested
15:22:06 css.1  | rules will be changing to match the behavior specified by CSS in an upcoming
15:22:06 css.1  | version. To keep the existing behavior, move the declaration above the nested
15:22:06 css.1  | rule. To opt into the new behavior, wrap the declaration in `& {}`.
15:22:06 css.1  | 
15:22:06 css.1  | More info: https://sass-lang.com/d/mixed-decls
15:22:06 css.1  | 
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/_type.scss
15:22:06 css.1  | 40  │       font-weight: $display-font-weight;
15:22:06 css.1  |     │       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ declaration
15:22:06 css.1  |     ╵
15:22:06 css.1  |     ┌──> node_modules/bootstrap/scss/vendor/_rfs.scss
15:22:06 css.1  | 136 │ ┌     @media (#{$rfs-mq-property-width}: #{$rfs-mq-value}) {
15:22:06 css.1  | 137 │ │       @content;
15:22:06 css.1  | 138 │ │     }
15:22:06 css.1  |     │ └─── nested rule
15:22:06 css.1  |     ╵
15:22:06 css.1  |     node_modules/bootstrap/scss/_type.scss 40:5            @import
15:22:06 css.1  |     bootstrap/scss/bootstrap.scss 17:9                     @import
15:22:06 css.1  |     app/assets/stylesheets/application.bootstrap.scss 1:9  root stylesheet
15:22:06 css.1  | 
15:22:10 css.1  | Error: Can't find stylesheet to import.
15:22:10 css.1  |   ╷
15:22:10 css.1  | 2 │ @import 'bootstrap-icons/font/bootstrap-icons';
15:22:10 css.1  |   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
15:22:10 css.1  |   ╵
15:22:10 css.1  |   app/assets/stylesheets/application.bootstrap.scss 2:9  root stylesheet
15:22:10 css.1  | error Command failed with exit code 65.
15:22:10 css.1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
15:22:10 css.1  | error Command failed with exit code 65.
15:22:10 css.1  | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
15:22:10 css.1  | [nodemon] app crashed - waiting for file changes before starting...
^C15:25:47 system | SIGINT received, starting shutdown
15:25:47 web.1  | - Gracefully stopping, waiting for requests to finish
15:25:47 web.1  | === puma shutdown: 2024-12-03 15:25:47 +0200 ===
15:25:47 web.1  | - Goodbye!
15:25:47 web.1  | Exiting
15:25:47 system | sending SIGTERM to all processes
15:25:47 web.1  | exited with code 0
15:25:47 js.1   | terminated by SIGINT
15:25:47 css.1  | terminated by SIGINT

```

### What operating system(s) are you seeing the problem on?

Linux

### What browser(s) are you seeing the problem on?

Firefox

### What version of Bootstrap are you using?

v5.3.3

**Comment by julien-deramond** [comment]
Closing as duplicate of https://github.com/twbs/bootstrap/issues/40962 (and other issues linked to Dart Sass)
